### ISSUE TITLE
search_fzf command causes crash when fzf not installed

### ISSUE THREAD

**Issue by jnlon** [kamiyaa/joshuto#211]
Hello. When I open joshuto and type 'S' without fzf installed, I get a hard crash/panic with the following stacktrace. When I install fzf I no longer experience the crash.

```
$ RUSTFLAGS=-g cargo build --release
    Finished release [optimized] target(s) in 0.14s

$ RUST_BACKTRACE=full ./target/release/joshuto
Failed to parse app config: No config directory found
Failed to parse keymap config: No config directory found
Failed to parse theme config: No config directory found
Failed to parse mimetype config: No config directory found
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/ui/backend.rs:70:32
stack backtrace:
   0:        0x10c4a83a2 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h408486b2bd61a14d
   1:        0x10c4c5e0a - core::fmt::write::h398bbe0d0a065ca9
   2:        0x10c492498 - std::io::Write::write_fmt::h9eb0f934b0343a80
   3:        0x10c497b58 - std::panicking::default_hook::{{closure}}::h9f1a676cea86d276
   4:        0x10c49780c - std::panicking::default_hook::he6707bf4997bbc04
   5:        0x10c4981c2 - std::panicking::rust_panic_with_hook::hd8e6eef88261a338
   6:        0x10c4a868a - std::panicking::begin_panic_handler::{{closure}}::hfba8e523b00f4966
   7:        0x10c4a84b7 - std::sys_common::backtrace::__rust_end_short_backtrace::h90514e9212607e52
   8:        0x10c497c93 - _rust_begin_unwind
   9:        0x10c4e1133 - core::panicking::panic_fmt::he4403f4ab5a63c4d
  10:        0x10c4e1017 - core::panicking::panic::h41037edb0b66fe2a
  11:        0x10c2c76e4 - core::option::Option<T>::unwrap::h2b642d8f99e29180
                               at /private/tmp/rust-20220923-6877-yszh4a/rustc-1.64.0-src/library/core/src/option.rs:775:21
  12:        0x10c2c76e4 - joshuto::ui::backend::AppBackend::terminal_ref::h27fdc0511330c65a
                               at /Users/jnlon/src/joshuto/src/ui/backend.rs:70:9
  13:        0x10c2c76e4 - joshuto::run::run_loop::h806fcdf2527362f7
                               at /Users/jnlon/src/joshuto/src/run.rs:47:27
  14:        0x10c2da065 - joshuto::run_main::h1cd022db1d6f0b82
                               at /Users/jnlon/src/joshuto/src/main.rs:117:9
  15:        0x10c2da065 - joshuto::main::h923327996a961164
                               at /Users/jnlon/src/joshuto/src/main.rs:178:11
  16:        0x10c2d04d6 - core::ops::function::FnOnce::call_once::haaec99057df26ed3
                               at /private/tmp/rust-20220923-6877-yszh4a/rustc-1.64.0-src/library/core/src/ops/function.rs:248:5
  17:        0x10c2d04d6 - std::sys_common::backtrace::__rust_begin_short_backtrace::h51bdf66aa8997642
                               at /private/tmp/rust-20220923-6877-yszh4a/rustc-1.64.0-src/library/std/src/sys_common/backtrace.rs:122:18
  18:        0x10c34f26c - std::rt::lang_start::{{closure}}::h967b89a598c3fba0
                               at /private/tmp/rust-20220923-6877-yszh4a/rustc-1.64.0-src/library/std/src/rt.rs:145:18
  19:        0x10c489fd5 - std::rt::lang_start_internal::h29747419ad3cbd79
  20:        0x10c2db319 - _main

```

### System Details

**OS**: macOS Monterey 12.6 (21G115)
**Hardware**: x86_64
**Joshuto Commit**: dabcffdc32f1bd5e7d074ea2006fc485e65fcd3e
**Rustc Version**: rustc 1.64.0

**Comment by kamiyaa** [comment]
Should be fixed via 36c93bb118ac16223c619e031b3f6b2ba4dbf9b8

**Comment by jnlon** [comment]
Can confirm it no longer crashes, thanks!
